clustered/client/src/main/java/org/ehcache/clustered/client/internal/config/xml/ClusteringServiceConfigurationParser.java

```diff
@@ -128,7 +129,19 @@ public URI getNamespace() {
             /*
              * <read-timeout> is an optional element
              */
-            getTimeout = processGetTimeout(fragment, item);
+            getTimeout = processTimeout(fragment, item);
```

Comment:
I'm thinking that this has now become complex enough that we should be thinking about a more robust parsing approach?  More JAXB?

Reply:
It is JAXB underneath

Reply:
no it is not, we do the parsing by hand for now. We should indeed consider moving to JAXB ... though that can (and should) be done in a separate PR.

Reply:
Fair enough.